### ISSUE TITLE
Anlegen / ändern von IP-Heizgruppen nicht möglich

### ISSUE THREAD

**Issue by Baxxy13** [jens-maus/RaspberryMatic#2105]
### Describe the issue you are experiencing

Seit 3.65.11.20221218, genauer seit Nightly 3.65.11.20221213 gibt es Probleme beim Erstellen / Ändern von IP-Heizgruppen,

Man kann:

- leere Gruppengeräte anlegen
- bestehende Gruppen (aus der Vorversion) löschen
- Gruppeneinstellungen (Geräteeinstellungen ändern)

Man kann nicht:

- Geräte zu einer frischen leeren oder bestehenden Gruppe hinzufügen
- Geräte aus einer bestehenden Gruppe löschen

### Describe the behavior you expected

Anlegen / Ändern von IP-Gruppen sollte wie gehabt funktionieren

### Steps to reproduce the issue

1. neue IP-Gruppe erzeugen
2. Gerät "hinzufügen" und "OK" drücken
3. 
...

### What is the version this bug report is based on?

3.65.11.20221218

### Which base platform are you running?

ova (Open Virtual Infrastructure)

### Which HomeMatic/homematicIP radio module are you using?

HmIP-RFUSB

### Anything in the logs that might be useful for us?

```txt
Keine Fehler in der Browserkonsole oder sonst wo sichtbar.
```

### Additional information

Nach Tests ist die neue `prototype.js` problematisch. Nach Austausch durch die Vorgänger-Version lassen sich die IP-Gruppen wieder fehlerfrei erzeugen.

Die neue `scriptaculous.js` hat aber Abhängigkeiten zur `prototype.js` so das an anderer Stelle (z.B. Posteingang) neue Fehler auftauchen wenn man die ältere `prototype.js` nutzt.

**Comment by jens-maus** [comment]
@Baxxy13 
Bitte mal folgende `prototype.js` version testen:

https://raw.githubusercontent.com/jens-maus/RaspberryMatic/master/buildroot-external/overlay/WebUI-raspmatic/www/webui/js/extern/prototype.js

Damit sollten die Probleme hoffentlich gelöst sein. Bitte aber auch testen ob die DVs danach auch existieren+funktionieren.

**Comment by Baxxy13** [comment]
Sieht soweit gut aus. 
Die DV's arbeiten auch.

Passt von meiner Seite.

**Comment by RunRudi** [comment]
Könntest du bitte einmal kurz eine Anleitung zum Update/Austausch der prototype.js geben?

**Comment by jens-maus** [comment]
Warum nicht einfach ein Downgrade auf die vorherige Version? Das wäre sicherlicher der bessere und DAU-sichere Weg. Aber wenn du unbedingt das angehen willst, dann einfach mal hier nachschauen. Da haben sich andere schon die Arbeit gemacht:

https://homematic-forum.de/forum/viewtopic.php?f=65&t=77164

**Comment by mikulik86** [comment]
Gibt es eine Möglichkeit, das HomeAssistant OS RaspberryMatic Add-On Downgraden?
Ich habe keine gefunden.

**Comment by jens-maus** [comment]
> Gibt es eine Möglichkeit, das HomeAssistant OS RaspberryMatic Add-On Downgraden? Ich habe keine gefunden.

Ja, in dem du das Backup zurückspielst das du natürlich gemacht hattest bevor du das Add-on geupdatet hattest...

**Comment by mikulik86** [comment]
Wie genau kann es gemacht werden?

**Comment by jens-maus** [comment]
> Wie genau kann es gemacht werden?

Das solltest du in der HomeAssistant Dokumentation nachschlagen können.

**Comment by mikulik86** [comment]
OK. 
Noch eine Frage: Was kann man tun wenn die aktuelle Version ist die erste Version die man in HA installiert hat?

**Comment by eNBeWe** [comment]
> OK. Noch eine Frage: Was kann man tun wenn die aktuelle Version ist die erste Version die man in HA installiert hat?

Ich hatte das gleiche Problem und habe mir mit dem aktuellen Snapshot-Build geholfen. Ich hoffe, dass ich beim nächsten Release dann einen Export der Daten aus dem Snapshot Addon machen kann, und das dann im nächsten Stable Release importieren kann.

**Comment by DerJunge3** [comment]
> @Baxxy13 Bitte mal folgende `prototype.js` version testen:
> 
> https://raw.githubusercontent.com/jens-maus/RaspberryMatic/master/buildroot-external/overlay/WebUI-raspmatic/www/webui/js/extern/prototype.js
> 
> Damit sollten die Probleme hoffentlich gelöst sein. Bitte aber auch testen ob die DVs danach auch existieren+funktionieren.

Guten Abend,
könntet Ihr mir netter Weise sagen unter welchem Pfad ich die Datei finde und austauschen muss ?
Vielen Dank und einen schönen Abend

**Comment by Baxxy13** [comment]
> unter welchem Pfad ich die Datei finde und austauschen muss ?

Folge einfach dem Link den Jens etwas weiter oben gepostet hat.
[2105#issuecomment](https://github.com/jens-maus/RaspberryMatic/issues/2105#issuecomment-1364332684)
